seven-days-to-die.kvp

```diff
@@ -37,7 +37,7 @@ App.HasReadableConsole=True
 App.SupportsLiveSettingsChanges=False
 App.LiveSettingChangeCommandFormat=set {0} "{1}"
 App.ApplicationIPBinding=0.0.0.0
-App.Ports=[{"Protocol":"Both","Port":26900,"Offset":0,"Range":1,"Ref":"ServerPort","Name":"Server and Steam Port","Description":"Main game traffic and Steam Query port","ChildPorts":null},{"Protocol":"TCP","Port":26901,"Offset":0,"Range":1,"Ref":"ControlPanelPort","Name":"Control Panel Port","Description":"Web panel access port","ChildPorts":null},{"Protocol":"UDP","Port":26902,"Offset":0,"Range":1,"Ref":"AdditionalPort","Name":"Server Additional Port","Description":"Unknown use at this time","ChildPorts":null},{"Protocol":"TCP","Port":27020,"Offset":0,"Range":1,"Ref":"RemoteAdminPort","Name":"RCON Port","Description":"RCON traffic port","ChildPorts":null}]
+App.Ports=[{"Protocol":"Both","Port":26900,"Offset":0,"Range":1,"Ref":"ServerPort","Name":"Server and Steam Port","Description":"Main game traffic and Steam Query port","ChildPorts":null},{"Protocol":"TCP","Port":8080,"Offset":0,"Range":1,"Ref":"ControlPanelPort","Name":"Control Panel Port","Description":"Web panel access port","ChildPorts":null},{"Protocol":"TCP","Port":8081,"Offset":0,"Range":1,"Ref":"RemoteAdminPort","Name":"RCON Port","Description":"RCON traffic port","ChildPorts":null}]
```

Comment:
I don't think it is a good idea configuring default ports that are in the AMP service port range

Reply:
It shouldn't have any impact. AMP will assign it like usual only using the application available ports.

Reply:
Acknowledge that. Still not a fan. And I don't rule out AMP doing something funky. Your call.

Reply:
I already made the changes if you want to review again.